drivers/clock_control/clock_stm32_ll_h5.c

```diff
@@ -435,7 +435,19 @@ static int set_up_plls(void)
 #endif
 
 #if defined(STM32_PLL_ENABLED)
+
+#if defined(CONFIG_BOOTLOADER_MCUBOOT)
```

Comment:
I think we rather need a specific config switch (or DT property) to tell Zephur to not touch the PLL.
To be enabled when the Zephyr image executes from an external flash clocked with that PLL.

Reply:
Don't you think this could be another PR sinc originally this one was for fixing the https://github.com/zephyrproject-rtos/zephyr/issues/87531

Reply:
see https://github.com/zephyrproject-rtos/zephyr/issues/92879